src/Npgsql/Internal/TypeHandlers/NumericHandlers/Int32Handler.cs

```diff
@@ -16,7 +17,8 @@ namespace Npgsql.Internal.TypeHandlers.NumericHandlers;
 /// </remarks>
 public partial class Int32Handler : NpgsqlSimpleTypeHandler<int>, INpgsqlSimpleTypeHandler<int>,
     INpgsqlSimpleTypeHandler<byte>, INpgsqlSimpleTypeHandler<short>, INpgsqlSimpleTypeHandler<long>,
-    INpgsqlSimpleTypeHandler<float>, INpgsqlSimpleTypeHandler<double>, INpgsqlSimpleTypeHandler<decimal>
+    INpgsqlSimpleTypeHandler<float>, INpgsqlSimpleTypeHandler<double>, INpgsqlSimpleTypeHandler<decimal>,
```

Comment:
If we allow int, shouldn't we also allow long and short enums?

Reply:
Sure, why not. It's just the original issue was only about int and string.